workspaces/meewee-time-registration/plugins/plugin-meewee-time-registration/meewee-time-registration/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@backstage-community/plugin-meewee-time-registration",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
You'll want to remove this or it won't get published to NPM.

Reply:
Done 👍